engines/leveldb/iter.go

```diff
@@ -46,11 +47,13 @@ func (i *ldbIterator) Key() []byte {
 }
 
 func (i *ldbIterator) Value() []byte {
-	obj, _ := i.Object()
-	if obj != nil {
+	obj, err := i.Object()
+	if err != nil {
+		fmt.Println(err)
+		return nil
+	} else {
```

Comment:
ah, yes, I was running into an issue where I wasn't marshalling/unmarshalling data correctly and the returned value was null with no indication of what went wrong. Since we can't return an error because of the interface is there a better way to provide the user with error information?